build-tools/firebase-deploy

```diff
@@ -173,7 +173,8 @@ make_pr_urls() {
 	# BASEURL_AI=https://evy-lang-stage--dev-bta34sbm.web.app
 	# BASEURL_DISCORD=https://evy-lang-stage-discord--dev-pap726z0.web.app
 	# BASEURL_DOCS=https://evy-lang-stage-docs--dev-62qb1rk7.web.app
-	# BASEURL_LEARN=https://evy-lang-stage-learn--dev-txkb2kn8.web.app
+	# BASEURL_LAB=https://evy-lang-stage-lab--dev-txkb2kn8.web.app
```

Comment:
Just double checking that lab was meant to get the suffix that was on learn, and that learn gets a new suffix, not lab.

Reply:
just demo URLs, but not making sense. Fixed.